src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -87,10 +87,10 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig, gdDAO: Datap
     }
   }
 
-  def listClusters(labelMap: Map[String, String]): Future[Seq[Cluster]] = {
+  def listClusters(labelMap: Map[String, String], includeDeleted: Boolean = false): Future[Seq[Cluster]] = {
```

Comment:
Instead, I could also just have 2 functions `listClusters(labelMap)` that gives all the clusters, and `listExistingClusters(labelMap)` that gives all clusters that aren't deleted.

Reply:
This is fine.